src/v1/authentication/__tests__/routers/authentication/login.js

```diff
@@ -0,0 +1,60 @@
+import simulate from '../../../../../utils/tests/requestHelper';
+import jwtService from '../../../../../services/jwtService';
+import mockDB from '../../../../users/testHelpers/mockDB';
```

Comment:
When I have the time, I'm gonna publish a db mapper that I use, I think it's gonna make it easier with the mock db data while testing

Reply:
so mentally prepare yourself that you're gonna have to change it soon :)

Reply:
Okay :)